pytest_selenium/drivers/testingbot.py

```diff
@@ -32,6 +33,12 @@ def key(self):
     def secret(self):
         return self.get_credential('secret', 'TESTINGBOT_SECRET')
 
+    def get_jenkins_credential(self, key):
+        if 'secret' == key:
+            return os.getenv(self.name.upper() + '_USR')
+        elif 'key' == key:
+            return os.getenv(self.name.upper() + '_PSW')
```

Comment:
Yeah, I was thinking that we should be consistent with key... but, your way makes more sense. :)